tests/framework/services/runner.sh.in

```diff
@@ -53,6 +53,8 @@ systemctl start postgresql-9.6
 sleep 5  # Unfortunately postgresql start seems to return before its truly up and ready for business
 su postgres -c 'createuser -R -S -d chroma'
 su postgres -c 'createdb -O chroma chroma'
+su postgres -c "psql -c 'CREATE EXTENSION IF NOT EXISTS btree_gist;' -d chroma"
```

Comment:
Does the user need to be specified here?

Reply:
Same as aboe